### PR TITLE
update + remove unnecessary param

### PR DESCRIPTION
**Pull request by teddyward**
- Updated the clients after pulling changes in VSO
- Removed unnecessary content: string parameter in all upload methods (already had contentStream: NodeJS.ReadableStream)

### CODE REVIEW

**Comment by teddyward** [comment]
When this gets merged tfs-cli will be broken until https://github.com/Microsoft/tfs-cli/pull/6 is merged

**Comment by bryanmacfarlane** [comment]
OK - so if you're breaking compat, then the right thing to do is increment the minor version.  When the tfx complementing PR goes in that needs to up it's consumption semver.  That ensures that any installs of tfx in that window will still work.  Swing by if the semver stuff isn't clear.  

Update this PR with the package ver bump and I'll look

**Comment by teddyward** [comment]
OK.  I think that we actually need to bump the major version to not break existing tfx installs since we used "vso-node-api": "^0.2.6" in the tfx package.json. As I understand it, the ^ will make it pull the most recent minor version... I guess I should change the '^' to '~' (which only bumps on patches) for the future

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "vso-node-api",
   "description": "Node client for Visual Studio Online/TFS REST APIs",
-  "version": "0.2.6",
+  "version": "0.3.6",
```
Comment (bryanmacfarlane) [inline package.json]: should 0.3.0

**Pull request closed without merging**